amazonlinux/content.md

```diff
@@ -0,0 +1,35 @@
+## What is Amazon Linux?
+
+[Amazon Linux](https://aws.amazon.com/amazon-linux-ami/) is a Linux-based operating system provided by Amazon Web Services. Amazon Linux is designed to provide a stable, secure, and high performance execution environment for applications running on Amazon EC2. It supports the latest EC2 instance features and includes packages that enable easy integration with AWS.
+
+Since 2011, Amazon Linux has been offered in the Amazon Machine Image (AMI) format. The Amazon Linux AMI is a supported and maintained image of Amazon Linux provided by AWS for use on EC2 instances.
+
+%%LOGO%%
+
+## Why should I use Amazon Linux container images?
+
+Existing customers that use Amazon Linux AMI in their EC2 instances can now use Amazon Linux for their containers. Customers now have a smooth path to use container technology for their development, testing and production no matter where they run.
+
+For customers just embarking on their container journey, the Amazon Linux container image is a great way to get started and is well integrated with AWS services like the EC2 Container Service (ECS) and ECR. Because the image is publicly available, you can use it to launch containers anywhere you need them -- from your laptop to ECS or anywhere in between.
```

Comment:
Ooh, does this ("from your laptop") mean that the Yum repos will be available outside EC2 as well? (IIRC they weren't when @yosifkit was testing the bits previously) :innocent: